### PR TITLE
added like filter info buttom and popup

### PR DESCRIPTION
**Pull request by pj-infest**

### CODE REVIEW

File: client/single_components/form_fields/drop_down.jsx [hunk 1]
```diff
@@ -4,11 +4,14 @@ import IconButton from 'material-ui/IconButton';
...
 import FlatButton from 'material-ui/FlatButton';
 import { _ } from 'meteor/underscore';
+import LikeFilterInfoBtn from './like_filter_info_btn';
 
 class DropDown extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { value: props.value || props.items[0].value };
+    this.state = {
+      value: props.value || props.items[0].value,
```
Comment (uapasha) [inline client/single_components/form_fields/drop_down.jsx]: as we are already here lets add `isMenuOpen` to the initial state as well

File: client/single_components/form_fields/drop_down.jsx [hunk 2]
```diff
@@ -62,22 +65,27 @@ class DropDown extends React.Component {
 
   render() {
     const { items, dropDownStyle, tooltip } = this.props;
+    const { openMenu } = this.state;
```
Comment (uapasha) [inline client/single_components/form_fields/drop_down.jsx]: lets call it `isMenuOpen`

File: client/single_components/form_fields/drop_down.jsx [hunk 3]
```diff
@@ -62,22 +65,27 @@ class DropDown extends React.Component {
...
+    const selectedValue = this.state.value;
```
Comment (uapasha) [inline client/single_components/form_fields/drop_down.jsx]: lets select it here:   `const { isMenuOpen, value: selectedValue } = this.state;`

File: client/single_components/form_fields/like_filter_info_btn.jsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+        example: "LIKE 'a_%_%'",
+        desc: "Finds any values that starts with 'a' and are at least 3 characters in length",
+      },
+      {
+        example: "LIKE 'a%o'",
+        desc: "Finds any values that starts with 'a' and ends with 'o'",
+      },
+    ];
+  }
+
```
Comment (uapasha) [inline client/single_components/form_fields/like_filter_info_btn.jsx]: lets move `style` and `tableData` to the constants outside of the class

File: client/single_components/form_fields/like_filter_info_btn.jsx [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+  handlePopverRequestClose() {
+    this.setState({
+      popoverOpen: false,
+    });
+  }
+
+  render() {
+    const { handlePopverTouchTap, handlePopverRequestClose, tableData } = this;
```
Comment (uapasha) [inline client/single_components/form_fields/like_filter_info_btn.jsx]: :+1:

File: client/single_components/form_fields/drop_down.jsx (hunk 1 above)
Comment (pj-infest) [inline client/single_components/form_fields/drop_down.jsx]: Done

File: client/single_components/form_fields/drop_down.jsx (hunk 2 above)
Comment (pj-infest) [inline client/single_components/form_fields/drop_down.jsx]: done

File: client/single_components/form_fields/drop_down.jsx (hunk 3 above)
Comment (pj-infest) [inline client/single_components/form_fields/drop_down.jsx]: Done

File: client/single_components/form_fields/like_filter_info_btn.jsx (hunk 4 above)
Comment (pj-infest) [inline client/single_components/form_fields/like_filter_info_btn.jsx]: Done

**Pull request merged**